app/views/pages/index.html.erb

```diff
@@ -1,9 +1,30 @@
-<div class="btn-group pull-right" data-toggle="buttons">
-    <% Filter.list.each do |filter| %>
-        <label id="<%= filter.filter_id %>" class="btn btn-primary" onclick="localStorage.setItem('jobfilter', '<%= filter.filter_id %>'); window.location = '?jobfilter=<%= filter.filter_id %>';" >
-          <input type="radio" name="<%= filter.filter_id %>" id="option_user" > <%= filter.title %>
-        </label>
-    <% end %>
+<div class="row pull-right">
+  <div class="div-inline dropdown">
+    <button class="btn btn-primary dropdown-toggle" type="button" id="dropdown-menu-filter" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
+      <span id="selected-filter-label"></span>
```

Comment:
You could probably set this in the ERB since you know what it should be because you captured them in `@jobfilter` and `@jobcluster`.

This will save you from having to set it in the Javascript on the page load.